src/screens/Customers/AddCustomer.tsx

```diff
@@ -78,26 +114,81 @@ function AddCustomer(props: AddCustomerProps) {
     setMeterNumber(event.target.value as string);
   }
 
+  const handleCustomerNumberInput = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setCustomerNumber(event.target.value as string);
+  }
+
   return (
     <BaseScreen title="Add New Customer" leftIcon="backNav">
-      <form noValidate className={classes.content} onSubmit={() => false}>
-        <TextField label={'Name'} id={'name'} onChange={handleNameInput} />
-        <FormControl variant="outlined" className={classes.formControl}>
-          <InputLabel id="select-tariff-plan-label">Select Tariff Plan</InputLabel>
-          <Select label={"Select Tariff Plan"} id={'select-tariff-plan'} labelId="select-tariff-plan-label" onChange={handleSelectTariffPlan}>
-            {tariffPlans.map((plan) =>
-              <MenuItem key={plan.id} value={plan.id}>{plan.name}</MenuItem>
-            )}
-          </Select>
-        </FormControl>
-        <Checkbox label={'Meter:'} textField={hasMeter} checkboxOnChange={() => setHasMeter(!hasMeter)} textFieldOnChange={handleMeterInput} />
-        <Checkbox label={'Customer is inactive'} checkboxOnChange={() => setCustomerInactive(!customerInactive)} />
-        <Button label={'Add'} onClick={handleSubmit} />
-      </form>
+      <BaseScrollView>
+        <form noValidate onSubmit={() => false}>
+          <div className={classes.textContainer}>
+            <div className={classes.inline}>
+              <div className={classes.nameInput}>
+                <TextField
+                  label={'Name'}
+                  id={'name'}
+                  placeholder={'e.g. Tom'}
+                  onChange={handleNameInput}
+                  required
+                />
+              </div>
+              <div className={classes.numberInput}>
+                <TextField
+                  label={'Number'}
+                  id={'number'}
+                  placeholder={'e.g. 12'}
+                  type="number"
+                  onChange={handleCustomerNumberInput}
+                  required
+                />
+              </div>
+            </div>
+          </div>
+          <div className={classes.selectContainer}>
+            <FormControl required variant="outlined" className={classes.formControl}>
+              <InputLabel>Meter Type</InputLabel>
+              <Select onChange={handleSelectMeterType}>
+                <MenuItem value={MeterType.ANALOG_METER}>Analog Meter</MenuItem>
+                <MenuItem value={MeterType.SMART_METER}>Smart Meter</MenuItem>
+                <MenuItem value={MeterType.NO_METER}>No Meter</MenuItem>
+              </Select>
+            </FormControl>
+          </div>
+          <div className={classes.textContainer}>
+            <TextField
+              label={'Meter Number'}
+              id={'meter-number'}
+              placeholder={'Input'}
+              type="number"
+              onChange={handleMeterNumberInput}
```

Comment:
I believe we want controlled inputs on all of our `TextField` (read the "Form Input" portion of Annie's description https://github.com/calblueprint/meepanyar/pull/87 ). Can you put `value={...}` for the textFields in this screen?

Reply:
Isn't it already value={props.value...} in TextField.tsx? I think this works because I can't type non-number stuff into type="number" inputs.